lib/beaker/host_prebuilt_steps.rb

```diff
@@ -505,10 +522,13 @@ def set_env host, opts
         host.exec(Command.new("touch #{host[:ssh_env_file]}"))
 
         #add the constructed env vars to this host
-        host.add_env_var('RUBYLIB', '$RUBYLIB')
-        host.add_env_var('PATH', '$PATH')
-        env.each_pair do |var, value|
-          host.add_env_var(var, value)
+
+        if host['platform'] !~ /freebsd/
```

Comment:
Without the updated env vars here you won't be able to run puppet commands - this is where the pe_defaults/foss_defaults are added to the host environment.